NagiosEnterprises/nagioscore
# Fix [#847]: Send a recovery notification if the object recovered while flapping

## To Test
```
define host {
    host_name               dummy_host
    address                 localhost
    max_check_attempts      2
    check_period            24x7
    notification_period     24x7
    contact_groups          admins
    check_command           check_dummy!0!Dummy
    check_interval          2
}

define service {
    host_name               dummy_host
    service_description     test_service
    check_command           check_dummy!0!Dummy
    max_check_attempts      2
    check_interval          2
    check_period            24x7
    contact_groups          admins
    notification_period     24x7
}
```
You can switch the check_dummy command between 2 and 3 until the host/service is flapping. Then make sure that you set it back to returning 0. When the host/service is no longer flapping, it should send a recovery notification like one would expect.
